target_chains/fuel/contracts/pyth-interface/src/data_structures/price.sw

```diff
@@ -175,6 +175,12 @@ impl PriceFeed {
         //convert publish_time from UNIX to TAI64
         publish_time += TAI64_DIFFERENCE;
 
+        // If a quorum of Wormhole Guardians were to be compromised or malicious they could sign a price update far out into the future (say u64::max()) , preventing any further updates since no new price can have a later `publish_time`, so we check here that the `publish_time` is not a future timestamp
+        require(
+            timestamp() >= publish_time,
+            PythError::PriceFeedPublishTimeInTheFuture,
+        );
```

Comment:
If the guardians get compromised they can really do anything (modify the price) and this prevention here won't have a significant impact.
This check is fine if the timestamp() on Fuel sequencer never falls behind. Maybe we ask Fuel guys before merging it in.